plenoptic/tools/data.py

```diff
@@ -132,12 +142,13 @@ def load_images(paths, as_gray=True):
                 # then multiple grayscales ones, so add channel dimension
                 images = images.unsqueeze(1)
     if images.ndimension() != 4:
-        raise Exception("Somehow ended up with other than 4 dimensions! "
-                        "Not sure how we got here")
+        raise Exception(
+            "Somehow ended up with other than 4 dimensions! " "Not sure how we got here"
+        )
     return images
 
 
-def convert_float_to_int(im, dtype=np.uint8):
+def convert_float_to_int(im: np.ndarray, dtype=np.uint8) -> np.ndarray:
```

Comment:
dtype here should use `Literal`

Reply:
Using `Union[np.uint8, np.uint16]` since Literal is for for builtin types.